README.md

```diff
@@ -1 +1,28 @@
 # 0Auth
+> A library for servicing using information stored in the user's local
+> area even if the data is not stored on the server.
+
+## Installation
+
+```bash
+// using npm
+npm install @0auth/server  // server library
+npm install @0auth/client  // client library
+
+// using yarn
+yarn add @0auth/server     // server library
+yarn add @0auth/client     // client library
+```
+
+## Usage
+
+should be written later.
```

Comment:
When adding the client and server example codes, a simple code example will be added together.